apps/frontend/src/app/5_pages/ProtocolDataPage/components/PricesBy24hChange/PricesBy24hChange.tsx

```diff
@@ -1,19 +1,191 @@
-import React, { FC } from 'react';
+import React, { FC, useCallback, useMemo, useState } from 'react';
 
 import { t } from 'i18next';
+import { Trans } from 'react-i18next';
+import { useNavigate } from 'react-router-dom';
 
-import { Paragraph } from '@sovryn/ui';
+import { getTokenDetailsByAddress } from '@sovryn/contracts';
+import {
+  Button,
+  ButtonStyle,
+  HelperButton,
+  Pagination,
+  Paragraph,
+  SimpleTableRow,
+  Table,
+} from '@sovryn/ui';
 
+import { AmountRenderer } from '../../../../2_molecules/AmountRenderer/AmountRenderer';
+import { AssetRenderer } from '../../../../2_molecules/AssetRenderer/AssetRenderer';
+import { useIsMobile } from '../../../../../hooks/useIsMobile';
+import { translations } from '../../../../../locales/i18n';
+import { decimalic } from '../../../../../utils/math';
 import { pageTranslations } from '../../ProtocolDataPage.constants';
+import { COLUMNS_CONFIG } from './PricesBy24hChange.constants';
+import { parseCryptoPairs } from './PricesBy24hChange.utils';
+import { PriceChange } from './components/PriceChange';
+import { useGetAssetData } from './hooks/useGetAssetData';
+import { useGetCryptoPairs } from './hooks/useGetCryptoPairs';
 
-export const PricesBy24hChange: FC = () => (
-  <div className="w-full md:bg-gray-90 md:py-7 md:px-6 rounded mb-9">
-    <Paragraph
-      className="md:text-2xl text-base font-medium"
-      children={t(pageTranslations.sections.pricesBy24hChange)}
-    />
-    <div className="min-h-72 mt-5">
-      <p>TODO: Add the actual content here</p>
+const pageSize = 8;
```

Comment:
I think its better to move the value to the constants file and name it something like DEFAULT_PRICES_PAGE_SIZE or similar